### ISSUE TITLE
🐞 Bug Report: goji --init provides an unexpected configuration file

### ISSUE THREAD

**Issue by VictorRos** [muandane/goji#7]
### 🐞 Describe the bug

Installed version `0.0.1-rc2`

- Run `goji --init`
- Content file is different from [here](https://github.com/muandane/goji/blob/main/.goji.json)
   - Properties are in lowercase
   - `entity` is not in documentation
   - Missing `"Types"`
   - Object not sorted

The generated file:

```json
{
  "Types": [
    {
      "emoji": "✨",
      "entity": "",
      "code": ":sparkles:",
      "description": "Introduce new features.",
      "name": "feature"
    },
    {
      "emoji": "🐛",
      "entity": "",
      "code": ":bug:",
      "description": "Fix a bug.",
      "name": "fix"
    },
    {
      "emoji": "📚",
      "entity": "",
      "code": ":books:",
      "description": "Documentation change.",
      "name": "docs"
    },
    {
      "emoji": "🎨",
      "entity": "",
      "code": ":art:",
      "description": "Improve structure/format of the code.",
      "name": "refactor"
    },
    {
      "emoji": "⚡️",
      "entity": "",
      "code": ":zap:",
      "description": "Improve performance.",
      "name": "perf"
    },
    {
      "emoji": "🚑️",
      "entity": "",
      "code": ":ambulance:",
      "description": "Critical hotfix.",
      "name": "hotfix"
    },
    {
      "emoji": "🚧",
      "entity": "",
      "code": ":construction:",
      "description": "Work in progress.",
      "name": "wip"
    },
    {
      "emoji": "📦",
      "entity": "",
      "code": ":package:",
      "description": "Add or update compiled files or packages.",
      "name": "package"
    }
  ],
  "Scopes": [
    "home",
    "accounts",
    "ci"
  ],
  "Symbol": true,
  "SkipQuestions": [],
  "SubjectMaxLength": 50
}
````

### 📖 Expected behavior

- Running `goji --init` should generate this [.goji.json](https://github.com/muandane/goji/blob/main/.goji.json)
- Properties are sorted on property `Name` in an ascending order

### 🌅 Screenshots

N/A

### 💻 Desktop

MacOS 13.3 (22E252)

**Comment by muandane** [comment]
my intention for the --init was to generate a more compact version of the example [.goji.json](https://github.com/muandane/goji/blob/main/.goji.json) and the users will customize their config file with [gitmoji](https://gitmoji.dev/).

Still i'll update to be as extensive as the example in [.goji.json](https://github.com/muandane/goji/blob/main/.goji.json)

**Comment by muandane** [comment]
fixes will be added to v0.0.1-rc3

**Comment by VictorRos** [comment]
Ok thank you, nice reactivity ;)
